src/coprocessor/endpoint.rs

```diff
@@ -804,6 +783,7 @@ mod tests {
         let mut worker = WorkerBuilder::new("test-endpoint").batch_size(30).create();
         let engine = engine::new_local_engine(TEMP_DIR, &[]).unwrap();
         let mut cfg = Config::default();
+        cfg.end_point_request_max_handle_duration = ReadableDuration::secs(0);
```

Comment:
Why is this set to zero?

Reply:
In the original test
```
let ctx = ReqContext {
            deadline: task.ctx.deadline
                - Duration::from_secs(super::DEFAULT_REQUEST_MAX_HANDLE_SECS),
```

this means that deadline is now (equals start_time) .

Reply:
Ok